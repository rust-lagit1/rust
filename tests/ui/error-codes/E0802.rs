#![feature(return_type_notation)]

trait Trait {}

fn test<T: Trait>()
where
    <T as Trait>::method(..): Send,
    //~^ ERROR associated function `method` not found for `T`
{
}

fn main() {}
