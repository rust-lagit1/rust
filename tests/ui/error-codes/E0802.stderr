error[E0220]: associated function `method` not found for `T`
  --> $DIR/E0802.rs:7:8
   |
LL |     T::method(..): Send,
   |        ^^^^^^ associated function `method` not found

error: aborting due to 1 previous error

For more information about this error, try `rustc --explain E0220`.
